ui/pages/management/index.js

```diff
@@ -1,27 +1,31 @@
 import { NoSsr } from "@material-ui/core";
 import Head from "next/head";
 import { withRouter } from "next/router";
-import React from "react";
+import React, { useEffect } from "react";
 import { connect } from "react-redux";
 import { bindActionCreators } from "redux";
 import MesheryPlayComponent from "../../components/MesheryPlayComponent";
 import { updatepagepath } from "../../lib/store";
-class Manage extends React.Component {
-  render() {
-    return (
-      <NoSsr>
-        <Head>
-          <title>Management | Meshery </title>
-        </Head>
-        <MesheryPlayComponent />
-      </NoSsr>
-    );
-  }
+
+function Manage(props) {
```

Comment:
Is Props are getting used? 

If not I can we can safely remove that.

Reply:
Sure, thanks